keanu-python/tests/test_sampling.py

```diff
@@ -64,6 +64,7 @@ def test_down_sample_interval(net: BayesNet) -> None:
     assert all(len(vertex_samples) == expected_num_samples for label, vertex_samples in samples.items())
 
 
+@pytest.mark.filterwarnings('ignore:Matplotlib is currently using agg')
```

Comment:
nit: pull the 'ignore:Matplotlib is currently using agg' out into a global if you'd like

Reply:
Ah yeah good call.

Reply:
I've added this to `pytest.ini` now. Take a look when you get a chance @GeorgeNash .